### DISCUSSION TITLE
Why is Master the default branch, when core is the updated branch?

### DISCUSSION THREAD

**Discussion by rdemint** [shuding/nextra#353] [Q&A]
Vercel will always use [the master branch](https://vercel.com/docs/concepts/git#production-branch), so it seems the future of the Nextra project will never be easily deployable on Vercel?  This seems strange to me. 

I wasted some time working in the master branch code before seeing the one-liner docs comment that core was the updated version with the features I was looking for. I'm still confused as the intended difference between the two branches. 

The Master branch deploys right to Vercel, great.  But the master branch does not have updates I need such as being able to change the GitHub icon, which has been updated in core branch.   

Perhaps I am misunderstanding the relationship between the two.  Someone please help me out. I would like to have the nextra ^2.0 running on Vercel.  How can I do this? 

Thanks,

**Comment by jermanuts** [comment]
The main branch is the old current nextra docs theme (nextra.vercel.app), nextra v2 docs which is more polished version of (swr.vercel.app) is in active development in the core branch.

For an instant vercel deploy from the core branch, you can https://vercel.com/new/clone?s=https://github.com/shuding/nextra/tree/core/docs

**Comment by dimaMachina** [comment]
`main` now is stable branch which points to nextra 3 now
